src/entities/user.entity.ts

```diff
@@ -25,4 +25,9 @@ export class User {
   beforeInsert() {
     this.password = bcrypt.hashSync(this.password, User.saltRounds);
   }
+
+  checkPassword(password: string) {
```

Comment:
I suggest you adding return type
```suggestion
  checkPassword(password: string): boolean {
```

Reply:
Resolved on edf90c596afd9a1a2857f1ecdbbf692b5cb750f8.